.travis.yml

```diff
@@ -21,10 +26,12 @@ install:
   - conda update -q conda
   - conda info -a
 
-  - conda env create -q -n skorch-env -f environment.yml python=${TRAVIS_PYTHON_VERSION}
+  - conda create -y -n skorch-env python=${TRAVIS_PYTHON_VERSION}
   - source activate skorch-env
-  - conda install --file=requirements-dev.txt
+  - pip install tabulate==0.7.7
+  - cat requirements.txt requirements-dev.txt > reqs.txt
+  - conda install --file=reqs.txt
   - python setup.py install
-  - conda install -c pytorch 'pytorch-cpu>=0.4.1'
+  - conda install -c pytorch pytorch-cpu==${PYTORCH_VERSION}
```

Comment:
Can you move tabulate to the `requirements.txt` instead of installing it here separately?

Reply:
Placing tabulate in `requirements.txt` would be preferable, but tabulate is not in conda. Thus we would not be able to use `conda install --file=reqs.txt` to install all the dependencies at once.

Reply:
There was an issue with this approach:  When pytorch gets installed, it will install the conda version of numpy. Having a pip version and a conda version of numpy installed may cause weird issues down the line.